app/pages/AgentInternalReferencePage.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import play.api.libs.json.JsPath
+
+case object AgentInternalReferencePage extends QuestionPage[String] {
+
+  override def path: JsPath = JsPath \ toString
```

Comment:
Can we scope this into an Agent object so the path will be `override def path: JsPath = JsPath \ Agent \ toString` where `Agent` is an object with its own path.